File: Measure-CS/ALL/Fibo/fibo.c
#include <stdio.h>
#include <stdlib.h>
int fibo(int n) {
	if (n < 2) {
		return n;
	}
	else return fibo(n-1) + fibo(n-2);
}
int main(int argc, char** argv) {
	printf("%d\n", fibo(atoi(argv[1])));
}

File: ALL/Fibo/fibo.py
def fibo(n):
	if n < 2:
		return n
	else:
		return fibo(n-2) + fibo(n-1)
def main(argv):
	print(fibo(int(argv[1])))
	return 0
if __name__ == '__main__':
	import sys
	main(sys.argv)
	
